### PR TITLE
fix error in videos without videoDetails #90

### PR DESCRIPTION
**Pull request by jhowbhz**
fix error in videos without videoDetails

### CODE REVIEW

**Comment by ytb2mp3** [comment]
hey @jhowbhz, thanks for the PR. Could you please downgrade the `?.` operators, so that older versions of NodeJS will be able to run this package as well?

File: lib/YoutubeMp3Downloader.js [hunk 1]
```diff
@@ -83,31 +83,31 @@ class YoutubeMp3Downloader extends EventEmitter {
...
          }
     
-        var videoTitle = this.cleanFileName(info.videoDetails.title);
+        var videoTitle = this.cleanFileName(info?.videoDetails?.title);
         var artist = 'Unknown';
         var title = 'Unknown';
-        var thumbnail = info.videoDetails.thumbnails ?
-            info.videoDetails.thumbnails[0].url
-            : info.videoDetails.thumbnail || null;
+        var thumbnail = info?.videoDetailsv.thumbnails ?
```
Comment (ytb2mp3) [inline lib/YoutubeMp3Downloader.js]: Is `videoDetailsv` a typo @jhowbhz?

Comment (jhowbhz) [reply lib/YoutubeMp3Downloader.js]: thanks, solved.

File: lib/YoutubeMp3Downloader.js [hunk 2]
```diff
@@ -83,12 +83,10 @@ class YoutubeMp3Downloader extends EventEmitter {
...
          }
     
-        var videoTitle = this.cleanFileName(info.videoDetails.title);
+        var videoTitle = typeof(info.videoDetails.title) != 'undefined' ? this.cleanFileName(info.videoDetails.title) : 'Unknown';
         var artist = 'Unknown';
         var title = 'Unknown';
-        var thumbnail = info.videoDetails.thumbnails ?
-            info.videoDetails.thumbnails[0].url
-            : info.videoDetails.thumbnail || null;
+        var thumbnail = info.videoDetailsv.thumbnails ? info.videoDetails.thumbnails[0].url : info.videoDetails.thumbnail || null;
```
Comment (ytb2mp3) [inline lib/YoutubeMp3Downloader.js:89]: I guess this also shouldn't be `info.videoDetailsv` instead of `info.videoDetails` @jhowbhz, right?

**Comment by jhowbhz** [comment]
Correct! thanks.

**Pull request closed without merging**